crypto/signer.go

```diff
@@ -0,0 +1,38 @@
+package crypto
+
+import (
+	"crypto/ecdsa"
+	"crypto/rand"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+)
+
+type Signer interface {
+	Sign(message []byte) (signed []byte, err error)
+}
+
+type EcSha256Signer struct {
+	key *ecdsa.PrivateKey
+}
+
+func NewEcSha256Signer(raw []byte) (s EcSha256Signer, err error) {
```

Comment:
```suggestion
func NewEcdsaSha256Signer(raw []byte) (s EcSha256Signer, err error) {
```
I think it's better to be specific.

Reply:
@feeblefakie 
Thank you for your advice.
I revised it in
https://github.com/scalar-labs/scalardl-go-client-sdk/pull/2/commits/7c69956b3401a4f7feb28c9d0aa9f6a26f7a9958

Please take a look when you have a chance.